OptimizelySDK/Entity/Audience.cs

```diff
@@ -49,7 +50,7 @@ public JToken ConditionList
                 if (DecodedConditions == null)
                 {
                     if (Conditions is string)
-                        DecodedConditions = Utils.ConditionEvaluator.DecodeConditions((string)Conditions);
+                        DecodedConditions = Utils.ConditionTreeEvaluator.DecodeConditions((string)Conditions);
```

Comment:
Please rename it with prefix _, private should be with _ at class level variable.